diff --git a/pylib/gyp/input.py b/pylib/gyp/input.py
index ca7ce44e..e7baa206 100644
--- a/pylib/gyp/input.py
+++ b/pylib/gyp/input.py
@@ -814,6 +814,8 @@ def ExpandVariables(input, phase, variables, build_file):
         # Find the ending paren, and re-evaluate the contained string.
         (c_start, c_end) = FindEnclosingBracketGroup(input_str[replace_start:])
 
+        print(":: [hoge] replace_start: {}, end: {}, end2: {}\n| c_start: {}, end: {}\n| input_str: {}\n| match: {}".format(replace_start, replace_end, replace_start + c_end, c_start, c_end, input_str, match))
+
         # Adjust the replacement range to match the entire command
         # found by FindEnclosingBracketGroup (since the variable_re
         # probably doesn't match the entire command if it contained
@@ -829,6 +831,8 @@ def ExpandVariables(input, phase, variables, build_file):
         contents_end = replace_end - 1
         contents = input_str[contents_start:contents_end]
 
+        print("| contents: {}, start: {}, end: {}".format(contents, contents_start, contents_end))
+
         # Do filter substitution now for <|().
         # Admittedly, this is different than the evaluation order in other
         # contexts. However, since filtration has no chance to run on <|(),
@@ -1077,6 +1081,7 @@ def ExpandVariables(input, phase, variables, build_file):
             output = (
                 output[:replace_start] + str(encoded_replacement) + output[replace_end:]
             )
+        print("| input_str: {}\n| output: {}".format(input_str, output))
         # Prepare for the next match iteration.
         input_str = output
 
@@ -1114,6 +1119,9 @@ def ExpandVariables(input, phase, variables, build_file):
     elif IsStrCanonicalInt(output):
         output = int(output)
 
+    print("L output(final?): {}".format(output))
+    if "iamdebugoption" in input_str:
+        traceback.print_stack()
     return output
 
 
@@ -1418,6 +1426,7 @@ def ProcessVariablesAndConditionsInList(the_list, phase, variables, build_file):
     index = 0
     while index < len(the_list):
         item = the_list[index]
+        #print(":: [fuga] index: {}\n| item: {}\n| the_list: {}".format(index, item, the_list))
         if type(item) is dict:
             # Make a copy of the variables dict so that it won't influence anything
             # outside of its own scope.
@@ -1426,6 +1435,9 @@ def ProcessVariablesAndConditionsInList(the_list, phase, variables, build_file):
             ProcessVariablesAndConditionsInList(item, phase, variables, build_file)
         elif type(item) is str:
             expanded = ExpandVariables(item, phase, variables, build_file)
+            if ("iamdebugoption" in item) or ("HOGE" in item) or ("FUGA" in item):
+                print(":: [fuga] index: {}\n| item: {}\n| the_list: {}".format(index, item, the_list))
+                print("L expanded: {}".format(expanded))
             if type(expanded) in (str, int):
                 the_list[index] = expanded
             elif type(expanded) is list:
