diff --git a/pylib/gyp/msvs_emulation.py b/pylib/gyp/msvs_emulation.py
index f744e38d..7b1bd3fa 100644
--- a/pylib/gyp/msvs_emulation.py
+++ b/pylib/gyp/msvs_emulation.py
@@ -84,8 +84,8 @@ def _AddPrefix(element, prefix):
     """Add |prefix| to |element| or each subelement if element is iterable."""
     if element is None:
         return element
-    # Note, not Iterable because we don't want to handle strings like that.
-    if isinstance(element, list) or isinstance(element, tuple):
+    if (isinstance(element, collections.Iterable) and
+            not isinstance(element, basestring)):
         return [prefix + e for e in element]
     else:
         return prefix + element
@@ -333,7 +333,7 @@ class MsvsSettings:
         # first level is globally for the configuration (this is what we consider
         # "the" config at the gyp level, which will be something like 'Debug' or
         # 'Release'), VS2015 and later only use this level
-        if self.vs_version.short_name >= 2015:
+        if self.vs_version.short_name >= "2015":
             return config
         # and a second target-specific configuration, which is an
         # override for the global one. |config| is remapped here to take into
@@ -537,7 +537,7 @@ class MsvsSettings:
                 )
             ]
         )
-        if self.vs_version.project_version >= 12.0:
+        if self.vs_version.project_version >= "12.0":
             # New flag introduced in VS2013 (project version 12.0) Forces writes to
             # the program database (PDB) to be serialized through MSPDBSRV.EXE.
             # https://msdn.microsoft.com/en-us/library/dn502518.aspx
