diff -ur origin/mozc/get-official-mozc.sh mod/mozc/get-official-mozc.sh
--- origin/mozc/get-official-mozc.sh	2021-02-19 22:10:38.000000000 +0900
+++ mod/mozc/get-official-mozc.sh	2021-05-26 02:33:07.161779105 +0900
@@ -1,8 +1,4 @@
 #!/bin/bash
 
-rm -rf mozc-master/
-wget -nc https://github.com/google/mozc/archive/master.zip
-unzip master.zip
-cp mozc-master/src/data/dictionary_oss/id.def .
-cat mozc-master/src/data/dictionary_oss/dictionary*.txt > mozcdic.txt
-rm -rf mozc-master/
+cp  mozc/src/data/dictionary_oss/id.def .
+cat mozc/src/data/dictionary_oss/dictionary{0*,10}.txt > mozcdic.txt
diff -ur origin/src/make-dictionaries.sh mod/src/make-dictionaries.sh
--- origin/src/make-dictionaries.sh	2021-05-24 00:10:04.000000000 +0900
+++ mod/src/make-dictionaries.sh	2021-05-26 02:32:38.798281961 +0900
@@ -3,14 +3,14 @@
 UTDICDATE="20210524"
 REVISION="1"
 
-#altcannadic="true"
-#edict="true"
+altcannadic="true"
+edict="true"
 jawikiarticles="true"
 jinmeiut="true"
 neologd="true"
 #nicoime="true"
-#skk="true"
-#sudachidict="true"
+skk="true"
+sudachidict="true"
 
 
 # ==============================================================================
@@ -98,15 +98,3 @@
 rm -f ../mozcdic*-ut-*.txt
 mv mozcdic-ut.txt.extracted ../mozcdic-ut-$UTDICDATE.$REVISION.txt
 
-
-# ==============================================================================
-# Make a mozcdic-ut package
-# ==============================================================================
-
-cd ../../
-rm -rf mozcdic-ut-$UTDICDATE.$REVISION
-rsync -av mozcdic-ut-dev/* mozcdic-ut-$UTDICDATE.$REVISION --exclude=id.def \
---exclude=jawiki-latest* --exclude=jawiki-ut.txt --exclude=KEN_ALL.* --exclude=*.csv \
---exclude=*.xml --exclude=*.gz --exclude=*.bz2 --exclude=*.xz --exclude=*.zip
-rm -f mozcdic-ut-$UTDICDATE.$REVISION/*/mozcdic*.txt*
-
diff -ur origin/edict/modify-edict2.rb mod/edict/modify-edict2.rb
--- origin/edict/modify-edict2.rb	2021-02-19 22:13:15.000000000 +0900
+++ mod/edict/modify-edict2.rb	2021-05-26 02:51:41.472317031 +0900
@@ -109,7 +109,7 @@
 # ==============================================================================
 
 `rm -f edict2`
-`wget -nc http://ftp.monash.edu/pub/nihongo/edict2.gz`
+`wget -nc http://ftp.edrdg.org/pub/Nihongo/edict2.gz`
 `gzip -dk edict2.gz`
 $filename = "edict2"
 $dicname = "mozcdic-edict2.txt"
