diff -ur origin/src/make-dictionaries.sh mod/src/make-dictionaries.sh
--- origin/src/make-dictionaries.sh	2021-12-05 12:35:05.000000000 +0900
+++ mod/src/make-dictionaries.sh	2021-12-09 22:09:26.143421139 +0900
@@ -16,12 +16,11 @@
 # Make each dictionary
 # ==============================================================================
 
-rm -f ../mozcdic-ut-*.txt
-rm -f ../*/mozcdic-*.txt*
-
 echo "Get Mozc and make mozc-*.tar.bz2..."
 cd ../mozc/
-ruby get_mozc_git.rb
+cp mozc/src/data/dictionary_oss/id.def .
+shopt -s extglob
+cat mozc/src/data/dictionary_oss/dictionary!(11).txt > mozcdic.txt
 
 echo "Get jawiki-titles and add search results to each title..."
 cd ../jawiki-titles/
@@ -119,27 +118,5 @@
 
 mv mozcdic-ut-$UTDICDATE.txt.need.costs ../mozcdic-ut-$UTDICDATE.txt
 
-echo "Update documents..."
-ruby update_documents.rb
-
-echo "Copy mozc-*.tar.bz2 and PKGBUILD..."
-cp -f ../mozc/mozc-*.tar.bz2 ../../
-cp -f ../pkgbuild/*.PKGBUILD ../../
-
-
-# ==============================================================================
-# Make mozcdic-ut-*.tar.bz2
-# ==============================================================================
-
-echo "Copy files to mozcdic-ut-$UTDICDATE..."
-cd ../../
-rm -rf mozcdic-ut-$UTDICDATE
-rsync -a mozcdic-ut-dev/* mozcdic-ut-$UTDICDATE --exclude=id.def --exclude=*.bzl \
---exclude=jawiki-latest* --exclude=jawiki-ut*.txt --exclude=KEN_ALL.* --exclude=*.csv \
---exclude=*.xml --exclude=*.gz --exclude=*.bz2 --exclude=*.xz --exclude=*.zip \
---exclude=*.html --exclude=_*.rb --exclude=*/mozcdic*.txt*
-
-echo "Compress mozcdic-ut-$UTDICDATE..."
-tar -cjf mozcdic-ut-$UTDICDATE.tar.bz2 mozcdic-ut-$UTDICDATE
-
-echo "Done."
+#echo "Update documents..."
+#ruby update_documents.rb
